scripts/fuzz_opt.py

```diff
@@ -1643,6 +1656,9 @@ def get_random_opts():
                 print('avoiding --flatten due to multivalue + reference types not supporting it (spilling of non-nullable tuples)')
                 print('TODO: Resolving https://github.com/WebAssembly/binaryen/issues/4824 may fix this')
                 continue
+            if '--enable-exception-handling' in FEATURE_OPTS:
+                print('avoiding --flatten due to exception-handling not supporting it (requires blocks with results)')
+                continue
```

Comment:
This is going to disable `--flatten` even for the old EH, but unless we give the new EH a separate feature flag, which I'm not sure we should do, we can't help, I guess..?

Reply:
Good catch @aheejin , I missed that. But it seems reasonable to do, and not worth adding a new feature flag.